scripts/shell/requirements2.txt

```diff
@@ -0,0 +1 @@
+nrfutil
```

Comment:
nrfutil is going to be python 3 only very soon:

https://github.com/NordicSemiconductor/pc-nrfutil/issues/275#issuecomment-545335205

So I'm confused about why we're moving this particular dependency to our python 2 requirements file.

Reply:
Following up on this after some offline discussion with the nrfutil team regarding that issue:

- they are indeed going python 3, but it will actually be python 3.7 *only* (not 3.6, not 3.8, but exactly python 3.7)
- this means we should absolutely not have it in our top-level requirements.txt since we try to support as many python versions as Zephyr does (currently 3.4+)
- so I understand and agree that it makes sense to do this move, but since the python 3 nrfutil release is right around the corner, perhaps we can name this `scripts/shell/requirements.txt`, and document that for now this is python 2 but it will soon be python 3.7 only?

I don't think the nrfutil team is going to be able to support many versions of Python in the near future, so we may want to consider limiting our dependence on it in general-purpose NCS dependencies.

cc: @carlescufi

Reply:
Just saw this got a +1 on the documentation side. What are we going to do about the nrfutil question?

I noticed they have since put out another prerelease for 6.0, which will be python 3 only:

https://github.com/NordicSemiconductor/pc-nrfutil/releases

We could merge this as-is since the stable release is python 2 only, and update the documentation later once python 3 is available.

Reply:
nrfutil 6.0 is out: https://github.com/NordicSemiconductor/pc-nrfutil/issues/275#issuecomment-559777455

Reply:
However, note that it requires python 3.6+, while zephyr still claims support for 3.4+.